docs/UserGuide.md

```diff
@@ -255,11 +251,11 @@ In this section, you will find the commands that are available on the main page.
 
 You can use this command to **create a new budget**.
 
-When a new budget is created it will be stored inside NUSave, and it will be reflected immediately in the list of budgets
-on the main page of NUSave. This command is only available when no budget is open.
+When a new budget is created, you will see the new budget displayed as an entry on the current page.
```

Comment:
Should it be "When you create a new budget, it will be displayed as a budget card in the list view."?